content/blog/enable-disable-middleware.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "Compile Time Enabling or Disabling Middleware"
+description = "Enable or Disable middleware when compiling CoreDNS"
+tags = ["Documentation"]
+draft = false
+date = "2017-07-25T16:07:39+01:00"
+author = "miek"
++++
+
+CoreDNS' [middleware](/middleware) (or [external middleware](/exmiddleware)) can be enabled or
+disabled on the fly by (not) specifying it in the [Corefile](/2017/07/23/corefile-explained/).

```

Comment:
I think:

on the fly by specifying (or not specifying) it in the

is easier to read.